zcash_client_backend/Cargo.toml

```diff
@@ -214,6 +214,14 @@ unstable-serialization = ["dep:byteorder"]
 ## Exposes the [`data_api::scanning::spanning_tree`] module.
 unstable-spanning-tree = []
 
+## Exposes access to the lightwalletd server via TOR
+tor-lightwalletd-tonic = [
+  "tor",
+  "lightwalletd-tonic",
+  "tonic?/tls",
+  "tonic?/tls-webpki-roots"
```

Comment:
I think this new feature is likely the wrong fix, and these two extra feature flags probably actually need to be part of the `tor` feature flag (because the currency conversion API uses HTTPS which requires TLS and the WebPKI roots as well).